linux_os/guide/system/logging/package_rsyslog_installed/rule.yml

```diff
@@ -29,6 +29,7 @@ references:
     isa-62443-2009: 4.3.3.3.9,4.3.3.5.8,4.3.4.4.7,4.4.2.1,4.4.2.2,4.4.2.4
     isa-62443-2013: 'SR 2.10,SR 2.11,SR 2.12,SR 2.8,SR 2.9'
     iso27001-2013: A.12.4.1,A.12.4.2,A.12.4.3,A.12.4.4,A.12.7.1
+    ospp: FTP_ITC_EXT.1.1
```

Comment:
The references must be in alphabetical order. Please place this new reference below the NIST ones. [utils/fix_rules.py](https://complianceascode.readthedocs.io/en/latest/manual/developer/05_tools_and_utilities.html#utils-fix-rules-py-automatically-fix-up-rules) can also help you fix this automatically.

Reply:
Sorry about that, my collation fu has failed me. Fixed in 9e022bb1f2c612fe436c140003af7c11e1620364.